config.js

```diff
@@ -44,6 +44,10 @@ const CONFIG = {
   // fetching again.
   SETTINGS_CACHE_TIME: ms(process.env.TALK_SETTINGS_CACHE_TIME || '1hr'),
 
+  // ALLOW_NO_LIMIT_QUERIES enables some queries to specify a limit of -1 to
+  // request all of the records. Otherwise, minimum limits of 0 are enforced.
+  ALLOW_NO_LIMIT_QUERIES: process.env.TALK_ALLOW_NO_LIMIT_QUERIES === 'TRUE',
```

Comment:
Use case? This looks kinda dangerous and slow. What can't be solved using subsequent requests with a limit?